packages/dds/tree/src/core/tree/anchorSet.ts

```diff
@@ -715,7 +713,11 @@ export class AnchorSet implements Listenable<AnchorSetRootEvents>, AnchorLocator
 			pathVisitors: new Map<PathNode, Set<PathVisitor>>(),
 			parentField: undefined as FieldKey | undefined,
 			parent: undefined as UpPath | undefined,
-			bufferedEvents: [] as { node: PathNode; event: keyof AnchorEvents }[],
+			bufferedEvents: [] as {
+				node: PathNode;
+				event: keyof AnchorEvents;
+				changedField?: FieldKey;
```

Comment:
What does it mean for this to be undefined? Might be worth adding an explicit type for this so we can more easily document the properties.